pep-0684.rst

```diff
@@ -0,0 +1,640 @@
+PEP: 684
+Title: A Per-Interpreter GIL
+Author: Eric Snow <ericsnowcurrently@gmail.com>
+Discussions-To: python-dev@python.org
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 08-Mar-2022
+Python-Version: 3.11
+Post-History: 08-Mar-2022
+Resolution:
+
+.. XXX Split out an informational PEP with all the relevant info,
+   based on the "Consolidating Runtime Global State" section?
+
+Abstract
+========
+
+Since Python 1.5 (1997), CPython users can run multiple interpreters
+in the same process.  However, they have always shared a significant
```

Comment:
```suggestion
in the same process.  However, these interpreters have always shared a significant
```

Clarify ambiguity with "they" naively referring to "users".

Reply:
FYI, the indirect pronoun already refers to the last noun used (that matches).  The last plural noun is "interpreters".

That said, I don't mind making it more clear. 🙂

Reply:
```suggestion
in the same process.  However, interpreters in the same process
have always shared a significant
```